index.js

```diff
@@ -0,0 +1,83 @@
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const add = document.querySelector('.add');
+const booksDiv = document.querySelector('.booksDiv');
+
+let arrayOfBooks = [];
+
+if (localStorage.getItem('books')) {
+  arrayOfBooks = JSON.parse(localStorage.getItem('books'));
+}
+function addElementsToPageFrom(arrayOfBooks) {
+  booksDiv.innerHTML = '';
+
+  arrayOfBooks.forEach((book) => {
+    const div = document.createElement('div');
+    div.className = 'book';
+
+    div.setAttribute('data-id', book.id);
+    const title = document.createElement('p');
+    title.appendChild(document.createTextNode(book.title));
+    div.appendChild(title);
+    const author = document.createElement('p');
+    author.appendChild(document.createTextNode(book.author));
+    div.appendChild(author);
+    // Create Delete Button
+    const span = document.createElement('button');
```

Comment:
[OPTIONAL]
- [ ] To be more specific you may consider using the descriptive name for your element for your code maintainability.
`AddBookButton` would sound more descriptive than `span` wouldn't you tell? :smile: